### PR TITLE
registry: Extend mirroring current Snowbridge API

### PR DESCRIPTION
**Pull request by NunoAlexandre**
❗  We should double check all the data added here

### CODE REVIEW

File: app/src/services/transfer.ts [hunk 1]
```diff
@@ -40,8 +40,8 @@ export const getErc20TokenContract = (
   env: Snowbridge.environment.SnowbridgeEnvironment,
 ): string => {
   switch (token.id) {
-    case 'weth':
-      return env.locations[0].erc20tokensReceivable[0].address
+    case 'weth' | 'veth':
+      return env.locations[0].erc20tokensReceivable[token.id].address
```
Comment (noahjoeris) [inline app/src/services/transfer.ts:44]: Here seems to be an error.
```suggestion
    case 'weth':
    case 'veth':
      return env.locations[0].erc20tokensReceivable[NUMBER].address // TODO: this now takes a number and not a string. So we need to change it
```

Comment (noahjoeris) [reply app/src/services/transfer.ts:44]: So for WETH it should be index = 0

Comment (NunoAlexandre) [reply app/src/services/transfer.ts:44]: I have a better fix for this 👍

**Review by noahjoeris — APPROVED** [review]
🚀

**Pull request merged**